packages/@webex/contact-center/src/cc.ts

```diff
@@ -0,0 +1,33 @@
+import {WebexPlugin} from '@webex/webex-core';
+import {CCConfig, IContactCenter, WebexSDK} from './types';
+import {WCC_API_GATEWAY} from './constants';
+
+export default class ContactCenter extends WebexPlugin implements IContactCenter {
+  namespace = 'WebexCC';
+  $config: CCConfig;
+  $webex: WebexSDK;
+  wccAPIURL: string;
+
+  constructor(...args) {
+    super(...args);
+    // @ts-ignore
+    this.$config = this.config;
+    // @ts-ignore
+    this.$webex = this.webex;
+  }
+
+  register(success: boolean): Promise<string> {
```

Comment:
Any reason we are resolving with a string?

Reply:
This is a dummy function which will be changed by Ravi in his PR

Reply:
Why not leave it as void then?

Reply:
We won't have anything for test files so added to test running test command using jest with legacy tools